kong/db/dao/init.lua

```diff
@@ -1481,6 +1483,15 @@ local function get_cache_key_value(name, key, fields)
 end
 
 
+local function hash_by_length(str)
+  if str and #str > LMDB_MAX_KEY_SIZE then
+    return sha256(str)
+  else
```

Comment:
In Kong's code we should add a blank line before `else`.

Reply:
ok